Documentation/Diagnostics/PH2076.md

```diff
@@ -44,6 +44,19 @@ Fix the previous example with this code:
             Assert.IsTrue(!isDone);
         }
 ```
+
```

Comment:
BadTestMethod1 does not trip any MS analyzer!

Reply:
The Migration section shall also be removed then.

Reply:
Removed